test/integration/src/test/java/io/pravega/test/integration/ReaderGroupNotificationTest.java

```diff
@@ -65,9 +83,12 @@
 import static org.junit.Assert.assertTrue;
 
 @Slf4j
-public class ReaderGroupNotificationTest {
+public class ReaderGroupNotificationTest extends LeakDetectorTestSuite {
 
     private static final String SCOPE = "test";
+    private static final int NUM_SEGMENTS = 10;
```

Comment:
For these 2 fields you use `NUM_` to indicate the count yet for the next one you use the `_COUNT` suffix. Please be consistent.